openflow_input/bsn_tlv

```diff
@@ -482,9 +482,16 @@ struct of_bsn_tlv_vlan_pcp : of_bsn_tlv {
     uint8_t value;
 };
 
+enum ofp_bsn_strip_vlan_mode(wire_type=uint8_t) {
+    OFP_BSN_STRIP_VLAN_NONE = 0,
+    OFP_BSN_STRIP_VLAN_OUTER = 1,
+    OFP_BSN_STRIP_VLAN_OUTER_INNER = 2,
```

Comment:
I can understand OUTER. Does OUTER_INNER strip the first two tags?
We can have up to 3 tags now, it would be good to cover all situations:
 - Strip all VLAN (tool can't handle VLAN)
 - Strip outer (the one we added) to preserve original VLANs
 - Strip outer VLAN of the original packet
It might better to use a bit field? If a bit is set, the VLAN tag of the corresponding position is stripped. This way, we can express all possibilities.